src/helpers/queryDataParser.js

```diff
@@ -139,12 +139,25 @@ const parseCategories = (data, skipLastDivider) => {
 
 const parsePointsOfInterestAndTours = (data) => {
   const pointsOfInterest = parsePointOfInterest(data?.[QUERY_TYPES.POINTS_OF_INTEREST]);
-
   const tours = parseTours(data?.[QUERY_TYPES.TOURS]);
 
   return _shuffle([...(pointsOfInterest || []), ...(tours || [])]);
 };
 
+const parseCrossData = (data) => {
+  const eventRecords = parseNewsItems(data?.[QUERY_TYPES.EVENT_RECORDS]);
+  const newsItems = parseNewsItems(data?.[QUERY_TYPES.NEWS_ITEMS]);
+  const pointsOfInterest = parsePointOfInterest(data?.[QUERY_TYPES.POINTS_OF_INTEREST]);
+  const tours = parseTours(data?.[QUERY_TYPES.TOURS]);
+
+  return {
+    [QUERY_TYPES.EVENT_RECORDS]: eventRecords,
+    [QUERY_TYPES.NEWS_ITEMS]: newsItems,
+    [QUERY_TYPES.POINTS_OF_INTEREST]: pointsOfInterest,
+    [QUERY_TYPES.TOURS]: tours
+  };
+};
```

Comment:
why have you decided to create local consts?
could return directly and fit in 100 line length:

```
const parseCrossData = (data) => ({
  [QUERY_TYPES.EVENT_RECORDS]: parseNewsItems(data?.[QUERY_TYPES.EVENT_RECORDS]),
  [QUERY_TYPES.NEWS_ITEMS]: parseNewsItems(data?.[QUERY_TYPES.NEWS_ITEMS]),
  [QUERY_TYPES.POINTS_OF_INTEREST]: parsePointOfInterest(data?.[QUERY_TYPES.POINTS_OF_INTEREST]),
  [QUERY_TYPES.TOURS]: parseTours(data?.[QUERY_TYPES.TOURS])
});
```

Reply:
it is removed altogether now